beetmoverscript/src/beetmoverscript/task.py

```diff
@@ -6,8 +6,8 @@
 
 import arrow
 from mozilla_version.errors import PatternNotMatchedError
-from mozilla_version.gecko import FirefoxVersion
 from mozilla_version.maven import MavenVersion
```

Comment:
ℹ️ This is the only place in `scriptworker-scripts` where we import `MavenVersion`.